pyblish_bumpybox/plugins/nuke/collect_nuke_groups.py

```diff
@@ -57,7 +57,7 @@ def process(self, context):
                         instance.data["families"]
                     )
                 )[0]
-                instance.data["outputPath"] = os.path.join(
+                instance.data["output"] = os.path.join(
```

Comment:
In fact, elsewhere in this PR you use drop cased key names, I think we should stick to a single convention and in my mind it should be the same as the naming used for variables.